### PR TITLE
Database should holde timezone information.

### LINKED ISSUE #205: Toggle "Created Invalid Date"
image

Wat

### PR DESCRIPTION
**Pull request by ivarconr**
closes #205

### CODE REVIEW

File: migrations/20170306233934-timestamp-with-tz.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(`
+ALTER TABLE events ALTER COLUMN created_at TYPE TIMESTAMP WITH TIME ZONE;
```
Comment (SimenB) [inline migrations/20170306233934-timestamp-with-tz.js]: How is timezone not default? So weird (timestamps should always be epoch (milli)seconds IMO)

**Comment by SimenB** [comment]
Unleash/unleash-frontend#71 for reference

File: migrations/20170306233934-timestamp-with-tz.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ALTER TABLE client_applications ALTER COLUMN seen_at TYPE TIMESTAMP WITH TIME ZONE;
+ALTER TABLE client_instances ALTER COLUMN created_at TYPE TIMESTAMP WITH TIME ZONE;
+ALTER TABLE client_instances ALTER COLUMN last_seen TYPE TIMESTAMP WITH TIME ZONE;
+ALTER TABLE client_metrics ALTER COLUMN created_at TYPE TIMESTAMP WITH TIME ZONE;
+       `, callback);
+};
+
+
+exports.down = function (db, callback) {
+    callback();
```
Comment (theneva) [inline migrations/20170306233934-timestamp-with-tz.js]: Such useful!

Comment (ivarconr) [inline migrations/20170306233934-timestamp-with-tz.js]: do not want to roll this back.

**Pull request merged**